templates/plugin/ansible.cfg

```diff
@@ -0,0 +1,4 @@
+[defaults]
+callback_whitelist = foreman
+local_tmp = /tmp
```

Comment:
oh, actually it is used in the other template, but i think it makes sense to use the same.